### ISSUE TITLE
Add `https://support.planetcom.co.uk` wiki pages.

### ISSUE THREAD

**Issue by shymega** [shymega/planet-devices-wiki-prs#21]
Currently a WIP, but I've managed to extract most of the Planet Computers' official support wiki pages, and will be merging into this wiki once I've tidied things up.

**Comment by ZimbiX** [comment]
As I DM'd, I'm not sure this effort is advisable.

I believe maintaining duplicated content to be an unnecessary burden (although updates to Planet's wiki will probably be extremely infrequent, given their track record of late).

I know you want this wiki to supplant the official one, but I'd only recommend doing this once Planet Computers actually agree to that, or at least agree to having a user-editable one.

If you want to go down the duplication path, it'd probably be a good idea to have a system in place to make it easier to know about what changed in the original wiki - e.g. having CI maintain a cache of the original's HTML files, perform diffs, and notify you if there's a change.

**Comment by shymega** [comment]
I don't think this is too much of an issue for Planet. They're aware of this wiki, and I've already copied their wiki into a branch.

The problem is, they could go bust, and we need to duplicate the firmware hosted there for future usage. CI wouldn't.. really work, I had to do some cURL hackery, and then delete content that was stub pages, etc. It's just not ideal to do that with CI. Especially with HTML sources.

I'd like to get other community thoughts on this effort as well - so it's not just us two :)
